examples/pom_file_generation/README.md

```diff
@@ -0,0 +1,70 @@
+# Generating `<dependency>` nodes for a POM file
+
+This example shows how you can generate the `<dependencies>` section of a POM
+file using `rules_jvm_external`'s generated targets and the `pom_file` rule from
+`bazel-common`.
+
+For reference,
```

Comment:
I don't get why you're showing how to generate a pom file for a library that already exists, and I found it distracting. As a user, I don't particularly care about the pom files of my dependencies - I care about my own pom file. Once I see aopalliance I start wondering what that is and why I care

Reply:
Good point, removed.